a_Calculate_Centers.R

```diff
@@ -4,26 +4,36 @@ tar_source("a_Calculate_Centers/src/")
 
 # Calculate POI Centers for all US and Territories Lakes -------------
 
-# This {targets} list calculates "Point of Inaccessibility", also known as
-# Cheybyshev Center for all lakes/reservoirs/impoundments greater than 1ha in 
-# surface area using the NHDPlusHR polygons using the {nhdplusTools} package
-# and the `poi()` function in the {polylabelr} package. For all waterbodies in
-# Alaska, POI were calculated based on the NHD Best Resolution file for the entire
-# state because the NHDPlusHR is not complete for AK. **Note**: this group of
-# targets will take up to 4h to complete.
-
-# create folder structure
-suppressWarnings({
-  dir.create("a_Calculate_Centers/mid/")
-  dir.create("a_Calculate_Centers/multisurface/")
-  dir.create("a_Calculate_Centers/out/")
-  dir.create("a_Calculate_Centers/nhd/")
-})
-
+# This {targets} list calculates "Point of Inaccessibility", also known as Cheybyshev 
+# Center for all lakes/reservoirs/impoundments greater than 1ha in surface area 
+# using the NHDPlus polygons using the {nhdplusTools} package and the `poi()` 
+# function in the {polylabelr} package. At some point, this workflow will need
+# to be updated to the new USGS 3DHP data, but that isn't complete right now. 
+# Additionally, we are intentionally using NHDPlusV2 instead of NHDPlusHR because
+# of computational time within the scope of this workflow. For HUC4s that are not
+# included in the NHDPlusV2, we access the NHD Best Resolution product directly
+# from the NHD from the National Map url.
 
 # create list of targets to perform this task
 a_Calculate_Centers_list <- list(
-  # get {sf}s for all US states and territories from {tigris}
+  tar_target(
+    name = a_check_dir_structure,
```

Comment:
It looks like this name has the `a` prefix, but other targets do not. Could remove?

Reply:
Since other groups will likely have this same target (but with different prefixes) I'm going to leave as-is, but update all the other targets to include the `a_` prefix! Thanks for catching this!